code/modules/mapping/reader.dm

```diff
@@ -330,10 +387,10 @@
 	var/relative_y = first_column.ycrd
 	var/highest_y = relative_y + y_relative_to_absolute
 
-	if(!cropMap && highest_y > world.maxy)
+	if(!crop_map && highest_y > world.maxy)
 		if(new_z)
 			// Need to avoid improperly loaded area/turf_contents
-			world.increaseMaxY(highest_y, max_zs_to_load = z_offset - 1)
+			world.increaseMaxY(highest_y, max_zs_to_load = z_offset - z_lower)
```

Comment:
is this right? why would loading in 1 less z layer mean I don't want to increase maxy all the way
z_offset is the lowest z level right?

Reply:
no idea, just copying previous behaviour. (I'm fairly certain the value isnt even used for modifying maxx and maxy

Reply:
figured it out